src/components/TextInput/BaseTextInput.js

```diff
@@ -245,6 +249,7 @@ class BaseTextInput extends Component {
                                     />
                                     {this.props.secureTextEntry && (
                                         <Pressable
+                                            ref={this.eyeButton}
```

Comment:
Can you please do a little refactoring in this component to change the name of the button to something that indicates its purpose (style names need changed too)? I suggest something like `showPasswordButton`.

Reply:
Done.